forte/pipeline.py

```diff
@@ -798,6 +810,11 @@ def add(
             # This will ask the job to keep a copy of the gold standard.
             self.evaluator_indices.append(len(self.components))
 
+        if ref_name is None:
+            self._ref_names.append("unknown")
```

Comment:
Maybe we can consider appending `None` here? There could be cases (although rare), where users actually use `unknown` as the name.